nilearn/input_data/nifti_labels_masker.py

```diff
@@ -60,6 +60,10 @@ class NiftiLabelsMasker(BaseMasker, CacheMixin):
         If standardize is True, the time-series are centered and normed:
         their mean is put to 0 and their variance to 1 in the time dimension.
 
+    standardize_strategy : str, optional
+        This parameter sets how the signal gets normalized by signal.clean()
+        (z-scored or percent signal change).
+
```

Comment:
idem

Reply:
What does `idem` mean?

Reply:
Sorry, it means 'same thing' or, here 'same remark'. It is French, but since the word is actually latine, I would have expected it to be used in English too.